example-get-started-experiments/code/.github/workflows/deploy-model.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy model
+
+on:
+  push:
+    # When a new version is registered in Studio Model Registry
+    tags:
+      - "results/train=pool-segmentation@*"
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Manual version name'
+        required: true
+        type: string
+  
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  deploy-model:
+    environment: cloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - id: clean_version_name
+      run: echo "version=$(echo ${{ github.ref_name || github.event.inputs.version }} | sed 's/[^a-zA-Z0-9-]/-/g')" >> "$GITHUB_OUTPUT"
+
+    - uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: us-east-2
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
```

Comment:
I need some help from the @iterative/platform team (or someone more AWS savvy than me).

When I use:

```
      - uses: aws-actions/configure-aws-credentials@v1
        with:
          aws-region: us-east-2
          role-to-assume: arn:aws:iam::342840881361:role/SandboxUser
          role-duration-seconds: 43200
 ```
 
The sagemaker API fails with:


```
botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the CreateModel operation: Could not access model data at s3://dvc-public/remote/get-started-pools/files/md5/8c/1d4e43ef621114f9798f9301e98cb3. Please ensure that the role "arn:aws:iam::***:role/SandboxUser" exists and that its trust relationship policy allows the action "sts:AssumeRole" for the service principal "sagemaker.amazonaws.com". Also ensure that the role has "s3:GetObject" permissions and that the object is located in us-east-2. If your Model uses multiple models or uncompressed models, please ensure that the role has "s3:ListBucket" permission.
```

Reply:
Responded in private